web/util/session.ts

```diff
@@ -25,3 +32,48 @@ export const sessionStart = (
     sessionIndex > -1 ? minutesFromProgramStart(sessions, sessionIndex) : 0;
   return addMinutes(start, sessionStartOffset);
 };
+
+const getDuration = ({ session, duration, durationOverride }: ProgramSession) =>
+  durationOverride ?? duration ?? session?.duration ?? 0;
+
+/* Given all Programs, finds the Programs that contain the Session (matching on sessionId === _id).
+ * For each match, returns:
+ * - label: Program's associated Venue name (using the first entry in the Program's Venues array)
+ * - rawDate: the Session's date (based on the Session's start time)
```

Comment:
Actually I'm really confused to what `rawDate` is supposed to be now. It's not really a "date", it's a moment in time as a string on a particular ISO format). It seems to be the start time of the _program_? Is this just a bug?

Example: cross-functional-collaboration-for-structured-content

```
<time datetime="2022-05-24T15:00:00.000Z">Tuesday, May 24</time>
<time datetime="PT3H30M">12:30–4:00 PM PDT</time>
```

So the machine-readable info seems to be just the moment in time when the _program_ starts and the duration of the session. What would make more sense to me—as long as it's giving one moment in time and one duration—is to have the moment in time when the _session_ starts (Tuesday, May 24, 12:30 PM PDT), and its duration.

```
<time datetime="2022-05-24T19:30:00.000Z">Tuesday, May 24</time>
<time datetime="PT3H30M">12:30–4:00 PM PDT</time>
```

Reply:
This.. seems to have been a somewhat significant bug. Fixed in 9e7120c. Once the `jest` setup from #163 lands, I'd like to use that to write some tests for this. Good catch.